app/components/Option/index.js

```diff
@@ -27,18 +27,18 @@ const styles = StyleSheet.create({
   container: {
     backgroundColor: Colors.white,
     borderColor: Colors.secondaryBorder,
-    borderRadius: 8,
-    borderWidth: 1,
-    marginBottom: 16,
-    paddingHorizontal: 10,
-    paddingVertical: 20,
+    borderRadius: Spacing.xxSmall,
+    borderWidth: 2,
```

Comment:
This *might* want to live in the design system. Do you see this border being a repeated pattern?

```suggestion
    ...Outlines.secondaryBoarder
```

Reply:
Funny, @achen178 and I discussed this, but we thought it would be a little outside of this PR's scope. Cool to introduce that as a separate PR?

Reply:
> Funny, @achen178 and I discussed this, but we thought it would be a little outside of this PR's scope. Cool to introduce that as a separate PR?

Totally, it’s your pr!